src/guides/hyperlinks.md

```diff
@@ -2,13 +2,13 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use this method inside of `$description[]` and the `$addField[]` value field.
+You can use hyperlinks inside [`$description[]`](/src/bdscript/description.md), [`$addField[]`](/src/bdscript/addField.md) and in [webhook](/src//guides/webhooks.md) content/description.
```

Comment:
Relative links from guide articles to bdscript articles should look like `../bdscript/name.html`, and not `/src/bdcript/name.md`